lib/matplotlib/tests/test_transforms.py

```diff
@@ -988,7 +988,7 @@ def test_transformed_path():
                     atol=1e-15)
 
     # Changing the path does not change the result (it's cached).
-    path.points = [(0, 0)] * 4
+    path._vertices = [(0, 0)] * 4
     assert_allclose(trans_path.get_fully_transformed_path().vertices,
```

Comment:
@QuLogic do you remember what the intent here was? I see you added this test way back in https://github.com/matplotlib/matplotlib/pull/4920, but `points` isn't a current Path attribute so it's not doing anything. (I'm not clear on the history of when/if that switched to `._vertices`)

Reply:
I removed this test for now, as it's unclear to me why this behavior was desired (and all other tests are passing).

Reply:
I believe it should be `path._vertices[:] = [(0, 0)] * 4` (it should definitely be an array, not a list.) That appears to pass for me.

Reply:
Oops, I forgot to save before testing; it does fail that way as well.

I think the problem in that test is that it uses `Affine2D` as the transform. The caching occurs on the _non-affine_ part of the transform and for `Affine2D`, that's a no-op that returns the original. So changing the original also changes the cached copy.

But I'm no longer sure this is something we want to codify as desired behaviour in a test, or whether that was just an implementation detail.